### PR TITLE
feat(testing): #1617 added test steps for finding row

### LINKED ISSUE #1617: When testing items in a list, we need to find the row and check that row
For example:
- user
- repositories
- monitors
- pings
- last updated date

### PR DESCRIPTION
**Pull request by webkhushboo**
closes #1617 

### Notes

A summary of what was achieved in this PR

- [ ] Added test steps for finding row and validating text

### CODE REVIEW

**Comment by webkhushboo** [comment]
@eddiejaoude 
Its all passing . See the attached screenshot 
image

File: web/test/step_definitions/browser.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
-import { Given, Then } from 'cypress-cucumber-preprocessor/steps';
+import {
+  Given,
+  Then
+} from 'cypress-cucumber-preprocessor/steps';
 
 const url = Cypress.config('baseUrl');
+let row;
```
Comment (eddiejaoude) [inline web/test/step_definitions/browser.js]: Cypress recommends not to do this, but to use their `alias` functionality.. 

```typescript
.as('row')
```

https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Sharing-Context

File: web/test/integration/homepage/projects.feature [hunk 2]
```diff
@@ -5,27 +5,27 @@ Feature:  Display public projects on the homepage
...
     And there is a document "test-user-minimum" with the js "user-stats/user-minimum" in collection "userStats"
     When the "/" page is open
-    Then the text "Test public project with minimum data title" is in the element ".project__list__title"
-    And the text "0" is in the element ".mat-column-repository"
-    And the text "0" is in the element ".mat-column-monitors"
-    And the text "0" is in the element ".mat-column-pings"
-    And the text "test-user-minimum" is in the element ".mat-column-user"
-    And the text "1 second ago" is in the element ".mat-column-lastDate"
+    And find the row of text "Test public project with minimum data title"
+    Then the text "0" is in the row
```
Comment (eddiejaoude) [inline web/test/integration/homepage/projects.feature]: For this can we be even more specific and use the row as well as the cell?

Comment (webkhushboo) [reply web/test/integration/homepage/projects.feature]: ok

File: web/test/step_definitions/browser.js (hunk 1 above)
Comment (webkhushboo) [reply web/test/step_definitions/browser.js]: ok

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**